### PR TITLE
AJAX replacement

### PR DESCRIPTION
**Pull request by acabunoc**

### CODE REVIEW

**Comment by acabunoc** [comment]
Excellent! PR is here at #112 :) I'll test out the changes!

File: public/widgets/paper-badger-widget.js [hunk 1]
```diff
@@ -69,16 +83,12 @@ function showBadges(confIn){
 	var containerClass=(confIn["container-class"]) ? confIn["container-class"] : "badge-container";	
 	var doi=confIn["article-doi"];
 		
-	$.get( "http://badges-mozillascience-org.herokuapp.com/papers/"+doi+"/badges", function( data ) {
-	
-	var entryData=JSON.stringify(data);
+	callAjax("http://badges-mozillascience-org.herokuapp.com/papers/"+doi+"/badges", function( entryData ) {
```
Comment (acabunoc) [inline public/widgets/paper-badger-widget.js]: Can we change the url to https://badges.mozillascience.org/?

Comment (acabunoc) [inline public/widgets/paper-badger-widget.js]: This way, we can take advantage of the CDN / caching

Comment (Alicole) [inline public/widgets/paper-badger-widget.js]: Yes certainly please do, don't forget the callAJAX function call too.

**Comment by acabunoc** [comment]
This works great for me! Just the small change url change. I'll send a PR over with that :)

**Comment by acabunoc** [comment]
Hey @Alicole - I can't seem to be able to see the latest changes you mentioned here: https://github.com/Alicole/PaperBadger/pull/1#issuecomment-136784122

As soon as they're in, I'm good to merge!

**Comment by Alicole** [comment]
@acabunoc new commit now completed.

**Pull request merged**